code/game/gamemodes/cult/runes.dm

```diff
@@ -60,7 +60,7 @@ To draw a rune, use an arcane tome.
 		to_chat(user, "<span class='notice'>You carefully erase the [lowertext(cultist_name)] rune.</span>")
 		qdel(src)
 	else if(istype(I, /obj/item/nullrod))
-		user.say("BEGONE FOUL MAGIKS!!")
+		user.say(pick("BEGONE FOUL MAGIKS!!", "AVAST, HERESY!!", "I BANISH YOU!!", "HOLY POWER ABJURES YOU!!", "YOU WILL BE STOPPED!!", "DEATH TO THE ONE WHO DREW THIS!!", "NAR'SIE WILL FALL!!"))
```

Comment:
`"NAR'SIE WILL FALL!!"`

I think you should keep that generic, if the same thing is done with the Ratvar runes, or  have it check for rune type

Reply:
Ratvar runes are done differently in a different file and are destroyed by hand, not null rod ^_^

Those runes are specific to the cult mode but I believe anyone can destroy runes.